### PR TITLE
Ruby on Rails capstone project Budget app pull request 

### PR DESCRIPTION
**Pull request by theresetuyi**
## Ruby on Rails capstone project Budget app

- se ruby gems as software packages system.
- Install Ruby on Rails framework.
- Understand Rails RESTful design and router.
- Use controllers to handle requests and render empty views.
- Use params from browser request in a safe way.
- Use preprocessed html file with embedded Ruby code.
- Use layouts and templates for shared content.
- Use database migration files to maintain database schema.
- Use validations for models.
- Secure app from n+1 problems.
- Understand what ORM is.
- Write SQL queries with ActiveRecord.
- Set up associations between models.
- Build a webapp that requires the user to log in.
- Use devise gem for authentication.
- Limit access to webapp resources based on authorization rules.
- Analyze in writing why you have made a coding choice using one structure over another.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,197 @@
...
+- **Create transactions**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://railsproject-eahw.onrender.com  )
+- [Loomlink] (https://www.loom.com/share/04fc614fdd524fcd96484b2e41ab7100)
```
Comment (Lucas-Masaba) [inline README.md]: - [ ] In the video you recorded, you did not mention a piece of code that you found interesting or that you are proud of making. Please add this to your video.

File: spec/integration/categories_spec.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
```
Comment (Lucas-Masaba) [inline spec/integration/categories_spec.rb:1]: - [ ] Please add some more integration tests fo the other controllers. here I only see that of the categories controller.

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @theresetuyi  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Met most requirements :gear: 
- Descriptive README :scroll: 
- Written tests passing :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
## Hi @theresetuyi, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


### Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**